cloudbuild-version.yaml

```diff
@@ -0,0 +1,35 @@
+steps:
+- name: gcr.io/cloud-builders/gcloud
+  entrypoint: 'bash'
+  args: [ '-c', "gcloud secrets versions access latest --secret=github-token-access --format='get(payload.data)' | tr '_-' '/+' | base64 -d > token.txt" ]
+- name: gcr.io/cloud-builders/git
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    set -e
+    echo "Set the git config..."
+    git config user.name $_GITHUB_USER
+    git config user.email $_GITHUB_EMAIL
+    git remote set-url origin https://$_GITHUB_USER:$(cat token.txt)@github.com/dataform-co/dataform.git
+    export git_branch_name=npm_veriosn_$(cat version.bzl | grep DF_VERSION | awk '{ print $3 }' | sed "s/\"//g")
+    echo "Create new branch $git_branch_name..."
+    git checkout -b $git_branch_name
+    echo "Update version..."
+    ./scripts/update_version
+    git add version.bzl
+    git commit -m "Update the npm package version"
+    echo "Push changes to remote..."
+    git push origin $git_branch_name
+    echo $git_branch_name > git_branch_name.txt
```

Comment:
Let's put this in an actual bash script, same as the others!

Reply:
If we put this in a bash script GCB is unable to replace env variables such as  `$_GITHUB_USER`.

Reply:
Are you sure? If they're env varaibles, then they should be available in the whole bash context.

Alternatively, it can be passed in to the script as an arg.

It will make it much easier to modify and update the script in future, rather than having to run it in cloud build every time